vllm/model_executor/layers/linear.py

```diff
@@ -134,6 +134,37 @@ def apply(self,
         return F.linear(x, weight, bias)
 
 
+class WeightTieLinearMethod(UnquantizedLinearMethod):
```

Comment:
nit: maybe a better name would be `TiedWeightLinearMethod`

Reply:
done